apps/web/composables/useStructuredData/useStructuredData.ts

```diff
@@ -25,9 +27,29 @@ export const useStructuredData: useStructuredDataReturn = () => {
    * setSigleItemMeta()
    * ```
    */
-  const setSingleItemMeta: SingleItemMeta = (product: Product, categoryTree: CategoryTreeItem) => {
+  const setSingleItemMeta: SingleItemMeta = async (product: Product, categoryTree: CategoryTreeItem) => {
```

Comment:
no need to be async

Reply:
Also I would suggest renaming that function to setProductMetaData because we always talk of products in the app and sdk